From a9a31fbbc56f33bdb61cf7973fd6b50db5cbf7f7 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Thu, 5 Nov 2020 18:07:12 +0800
Subject: [PATCH] arm: dts: rk3288: remove regulator-xxx-microvolt for SWITCH
 pmic node

Switch is just a function of switching. There is no voltage setting
function. Voltage getting is the supply voltage.

Fixes: 3fc99e38fdbf ("regulator: resolve supply after creating
regulator")

Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
Change-Id: I22ff04b59b9b051052348420f25999dae424d4ac
---
 arch/arm/boot/dts/rk3288-evb-rk808-linux.dts | 4 ----
 arch/arm/boot/dts/rk3288-firefly-rk808.dts   | 2 --
 arch/arm/boot/dts/rk3288-vyasa.dts           | 4 ----
 3 files changed, 10 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288-evb-rk808-linux.dts b/arch/arm/boot/dts/rk3288-evb-rk808-linux.dts
index 9403e62b33ab..22b44e37aca2 100644
--- a/arch/arm/boot/dts/rk3288-evb-rk808-linux.dts
+++ b/arch/arm/boot/dts/rk3288-evb-rk808-linux.dts
@@ -410,8 +410,6 @@
 			vcc_sd: SWITCH_REG1 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
 				regulator-name = "vcc_sd";
 				regulator-state-mem {
 					regulator-on-in-suspend;
@@ -421,8 +419,6 @@
 			vcc_lcd: SWITCH_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
 				regulator-name = "vcc_lcd";
 				regulator-state-mem {
 					regulator-off-in-suspend;
diff --git a/arch/arm/boot/dts/rk3288-firefly-rk808.dts b/arch/arm/boot/dts/rk3288-firefly-rk808.dts
index 02b8ec3011b1..7b9401886484 100644
--- a/arch/arm/boot/dts/rk3288-firefly-rk808.dts
+++ b/arch/arm/boot/dts/rk3288-firefly-rk808.dts
@@ -527,8 +527,6 @@
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-name = "vcc_18";
-				regulator-min-microvolt = <1800000>;
-				regulator-max-microvolt = <1800000>;
 				regulator-state-mem {
 					regulator-on-in-suspend;
 				};
diff --git a/arch/arm/boot/dts/rk3288-vyasa.dts b/arch/arm/boot/dts/rk3288-vyasa.dts
index 4856a9fc0aea..aebed2c7505e 100644
--- a/arch/arm/boot/dts/rk3288-vyasa.dts
+++ b/arch/arm/boot/dts/rk3288-vyasa.dts
@@ -325,8 +325,6 @@
 
 			vcc_sd: SWITCH_REG1 {
 				regulator-name = "vcc_sd";
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-state-mem {
@@ -336,8 +334,6 @@
 
 			vcc_lan: SWITCH_REG2 {
 				regulator-name = "vcc_lan";
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-state-mem {
-- 
2.35.3

